app/views/events/finance.html.erb

```diff
@@ -13,7 +13,17 @@
         </tr>
         <% ed.timecard_entries.each do |entry| %>
           <tr>
-            <td><%= entry.member %> - <%= entry.eventpart.text %></td>
+            <td><%= entry.member %>
+                <% event_roles = @event.run_positions_for(entry.member) %>
+                <% ed_roles = ed.run_positions_for(entry.member) %>
+                <% if not event_roles.empty? and not ed_roles.empty? %>
+                  (<b><% event_roles.each do |pos| %><%= pos.try(:role) %><% end %></b>, 
+                   <i> <% ed_roles.each do |ed_pos| %> <%= ed_pos.try(:role) %><% end %></i>)
+                <% elsif not event_roles.empty? and ed_roles.empty? %>
+                  (<b><% event_roles.each do |pos| %><%= pos.try(:role) %><% end %></b>)
+                <% elsif event_roles.empty? and not ed_roles.empty? %>
+                  (<b><% ed_roles.each do |ed_pos| %><%= ed_pos.try(:role) %><% end %></i>)
```

Comment:
I think for style consistency for the finance person, it would be best if one type of role is always bold and the other is italicized. It might be good to add a key somewhere for that information or **even better just put these in two additional columns.**

Reply:
If the styles are consistent, then you can also probably have only one case in the `if` (since no items would be an empty list, but I guess irrelevant if in new columns).